msg/MotionPlanRequest.msg

```diff
@@ -51,6 +51,21 @@ float64 allowed_planning_time
 float64 max_velocity_scaling_factor
 float64 max_acceleration_scaling_factor
 
+# New joint limits to use when running time-parameterization and smoothing.
+# Note, this does not replace the values in the robot model.
+# Only applicable for TOTG and Ruckig
+# Set the 'has_limits' field to true for velocity, acceleration, and/or jerk
+# in order for that limit to be applied. Position limits are not supported.
+JointLimits[] joint_limits
+
+# If true, use the joint limits defined in the 'joint_limits' field.
+# Otherwise, use the max scaling factors defined for velocity and acceleration
+bool use_joint_limits
+
+# If true, smoothing will be skipped when applying a smoothing adapter during
+# trajectory processing (if it is defined as a request adapter).
+bool skip_smoothing
```

Comment:
This makes me thing that a separate post processing config would make sense by now, there are already so many related fields.

Reply:
Hmm, say TOTG and Ruckig are configured to do smoothing by default in `ompl_planning.yaml`. I'm trying to think of how the motion planning pipeline would know to skip both of those smoothers but not the other OMPL planning request adapters.

Maybe it should be a vector of strings containing the plugins to skip, like this:

`string[] ompl_adapters_to_skip`

So to skip those two plugins, you would have:

["default_planner_request_adapters/AddTimeOptimalParameterization"
 "default_planner_request_adapters/AddRuckigTrajectorySmoothing"]

Reply:
I'm not sure we'd want to skip a time-parameterization adapter like TOTG. I think this is only relevant for smoothing in which case I think ruckig is the only one currently. In any case though, if multiple smoothing adapters do exist down the road, then it's not too hard to use the "skip_smoothing" field when the adapter is called like I have done on my fork here for Ruckig https://github.com/mechwiz/moveit2/blob/mwiz/hp_fixes/moveit_ros/planning/planning_request_adapter_plugins/src/add_ruckig_traj_smoothing.cpp#L68

Reply:
bumping this again.
I feel like defining a vector of strings for this is maybe a little overkill. I can think of usecases where even though a smoothing adapter (like Ruckig) is applied after TOTG in the list of ompl adapters, there may be exceptions/times where we don't want to smooth the resulting trajectory (whether to save on time or because its a very simple motion like closing/opening a gripper, etc..). Are there other usecases where we'd want to skip an adapter that is not smoothing?